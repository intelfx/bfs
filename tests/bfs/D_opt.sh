bfs_diff -D opt -nohidden -not \( -type c -o -type d \) -links -5 -links -10 -not -hidden basic
