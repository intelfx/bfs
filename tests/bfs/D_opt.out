basic/a
basic/b
basic/c/d
basic/e/f
basic/j/foo
basic/k/foo/bar
basic/l/foo/bar/baz
